docker-images/kafka/kafka-thirdparty-libs/2.6.x/pom.xml

```diff
@@ -24,7 +24,7 @@
     <repositories>
         <repository>
             <id>jcenter</id>
```

Comment:
Maybe you can rename the `id` from `jcenter` to `cruisecontrol` or something like that to correspond to the repository?

Reply:
I've made the changes and pushed in new commit. Thanks!